### ISSUE TITLE
Meeting: AMANDA Permitting Metrics Requested by Jim D.

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#4414]
### 11/6/20 w/ Ryan M. & Paloma A
**Discuss a recent email request from Jim for AMANDA permitting metrics**

- Total time from application submittal to permit issuance 
- Number of review cycles
- Time in ATD's Hands & Time in Applicant's Hands
- Number & Type of submittal deficiencies

**Comment by TracyLinder** [comment]
## Notes

### Total time from application submittal to permit issuance
- _(Folder Indate - Folder IssueDate)_
- We will be able to provide this metric after #1909 & #1726 are in PROD

**Story:** I want to be able to query all ROW permits by date range and retrieve data that will allow me to see turn-around averages by folder & subtype, and also allows me to sort and filter through each record as needed. I also want an optional query parameter to query by the permit Facility Owner when one exists. 
- ATD Menu report; output Excel
- Columns & parameters TBD
- A menu report is fine; no need for a batch email. 
- [ ] Ask Sree if she can write a query for this that can be used while we wait for the upgrade 
- Query based on folder info Facility Owner value 
![image.png](https://images.zenhubusercontent.com/5cca1e522b8dfc0ee14dedf5/503701d2-3ca9-4503-85d4-d7e63294df7e)


-------


### Number of review cycles
- Paloma to brainstorm w/ staff to see if there is any interum solution to this outside of the formal review process update

Should we impliment a formal review process
------


### Time in ATD's Hands & Time in Applicant's Hands
This is the total time the ball was in ROW's court vs pending update from the customer. 

- Paloma to brainstorm w/ staff to see if there is any interum solution to this outside of the formal review process update

- Concern: customer responds to us via email, and it could take staff a week to update the system w/ the response... this won't allow us to capture the true metrics like how long it took us or the customer to repond to each other
---


### Number & Type of submittal deficiencies

submittal deficiencies for permitting for example, they need to provide a TCP, they need to coordinate w/ stakeholders & get back to us, so basically any time review is put on hold pending customer action.
